data/event/src/commonMain/kotlin/com/adammcneilly/pocketleague/data/event/StartGGApolloClient.kt

```diff
@@ -0,0 +1,9 @@
+package com.adammcneilly.pocketleague.data.event
+
+import com.apollographql.apollo3.ApolloClient
+
+val startGGApolloClient = ApolloClient
+    .Builder()
+    .addHttpHeader("Authorization", "Bearer 24a270722c5612eef2ddf509817672ce")
+    .serverUrl("https://api.start.gg/gql/alpha")
```

Comment:
Move this into a new `data:startgg` module and also remove this token & store in secrets.